cpp/src/barretenberg/stdlib/hash/sha256/sha256.bench.cpp

```diff
@@ -2,10 +2,15 @@
 #include <benchmark/benchmark.h>
 #include "barretenberg/ecc/curves/bn254/fr.hpp"
 #include "barretenberg/plonk/composer/ultra_composer.hpp"
-#include "barretenberg/stdlib/types/types.hpp"
+#include "barretenberg/plonk/proof_system/prover/prover.hpp"
+#include "barretenberg/stdlib/primitives/packed_byte_array/packed_byte_array.hpp"
 
 using namespace benchmark;
-using namespace proof_system::plonk::stdlib::types;
+using namespace proof_system::plonk;
+
+using LocalComposer = UltraComposer;
+using LocalProver = UltraProver;
+using LocalVerifier = UltraVerifier;
```

Comment:
What does "LocalComposer" mean here versus just "Composer" in `sha256.test.cpp`?

Reply:
Well, I couldn't use alias `Prover` because that clashes with something in the namespace `proof_system::plonk` so I called that `LocalProver` and it propagated that way. I was getting some weird linker error from Google benchmark but I nixed the `using` invocation now and added some qualifiers to achieve more consistent aliasing.